tox.ini

```diff
@@ -22,12 +24,7 @@ commands=
 [base]
 deps=
     -r{toxinidir}/requirements_test.txt
-    !{notebooks}: -r{toxinidir}/requirements_dev.txt
-    notebooks: nbclient
-    notebooks: nbconvert
-    notebooks: nbformat
-    notebooks: ipykernel
-    notebooks: ipython<8.17.1
+    -r{toxinidir}/requirements_dev.txt
```

Comment:
btw, you can just merge it into just using `requirements_dev.txt`

Reply:
done!